src/library/vm/vm_float.cpp

```diff
@@ -122,7 +122,7 @@ void initialize_vm_float() {
     DECLARE_VM_BUILTIN(name({"native", "float", "acosh"}), [](vm_obj const & a) {return mk_vm_float(std::acosh(to_float(a)));});
     DECLARE_VM_BUILTIN(name({"native", "float", "atanh"}), [](vm_obj const & a) {return mk_vm_float(std::atanh(to_float(a)));});
 
-    DECLARE_VM_BUILTIN(name({"native", "float", "pi"}),    [](){return mk_vm_float(M_PI);});
+    DECLARE_VM_BUILTIN(name({"native", "float", "pi"}),    [](){return mk_vm_float(3.14159265358979323846);});
```

Comment:
Try to include `<math.h>` instead of hard coding `pi`

Reply:
Didn't including <math.h> cause something else to fail?

Reply:
Yes, I remember replacing it with `cmath`. Let's include both to please Windows and Linux

Reply:
Maybe I am being naive: I feel like hardcoding pi is more robust. It's not like it's a parameter we are going to have to change later.

Reply:
I have visceral response to hard coding constants. It's a big software engineering no-no. One reason is that you may not notice if you got it wrong and changed digits. Another is that, if your source is wrong and gets corrected, it won't be reflected in your code. Yet another is that, by putting it in your code, you take responsibility for all the complex considerations that come into writing a floating point number one way instead of another. In general, it's safe to assume that library writers know their stuff and they have put a lot of thought into a lot of aspects of their code and trusting them blindly, although overly simplistic,  is still better than thinking "this doesn't look hard, I'll redo their work".